.github/workflows/props-bot.yml

```
@@ -0,0 +1,88 @@
+name: Props Bot
+
+on:
+    # This event runs anytime a PR is (re)opened, updated, marked ready for review, or labeled.
+    # GitHub does not allow filtering the `labeled` event by a specific label.
+    # However, the logic below will short-circuit the workflow when the `props-bot` label is not the one being added.
+    # Note: The pull_request_target event is used instead of pull_request because this workflow needs permission to comment
+    # on the pull request. Because this event grants extra permissions to `GITHUB_TOKEN`, any code changes within the PR
+    # should be considered untrusted. See https://securitylab.github.com/research/github-actions-preventing-pwn-requests/.
+    pull_request_target:
+        types:
+            - opened
+            - synchronize
+            - reopened
+            - labeled
+            - ready_for_review
+    # This event runs anytime a comment is added or deleted.
+    # You cannot filter this event for PR comments only.
+    # However, the logic below does short-circuit the workflow for issues.
+    issue_comment:
+        type:
```

You'll most likely want to update this to `types`. `type` is not supported:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#issue_comment

```suggestion
        types:
```

---

Thanks for the catch. Now to raise the fix over at https://github.com/WordPress/gutenberg/blob/trunk/.github/workflows/props-bot.yml#L21

---

Since the problem is the same in the Gutenberg repo and the WordPress-develop repos, I opened PRs to fix the issue there as well:

https://github.com/WordPress/gutenberg/pull/64557
https://github.com/WordPress/wordpress-develop/pull/7203